README.md

```diff
@@ -9,7 +9,12 @@ Our Hubot-based bot to automate lots of stuff including:
 
 # How to use it
 
-In your chat channel tag at start, middle or end of a message:
+To get `help` command, run the following code:
+`ourbot help` - to get general help message information
+
+`/ourbot help` - to see available functionalities of `ourbot`. 
+
+In your chat channel tag at start, middle or end of a message: `+todo`, `+standup`, `+examples`
```

Comment:
This should have a heading e.g. `## TODOs` so that it can be specifically linked to from the help